gen/field.go

```diff
@@ -454,6 +454,11 @@ func (f fieldGroupGenerator) Equals(g Generator) error {
 		//
 		// This function performs a deep comparison.
 		func (<$v> *<.Name>) Equals(<$rhs> *<.Name>) bool {
+			if <$v> == nil {
```

Comment:
optional:
```
if <$v> == nil || <$rhs> == nil {
  return <$v> == <$rhs>
}
```
would have the same logic in less lines, but maybe less readable.

Reply:
Hah, that was what I originally had. I decided more readable was preferable to
fewer lines.